docs/src/main/sphinx/connector/jdbc-common-configurations.fragment

```diff
@@ -33,3 +33,10 @@ connector:
       Do not change this setting from the default. Non-default values may
       negatively impact performance.
     - 1000
+  * - ``join-pushdown.enabled``
+    - Enable join pushdown. Join pushdown is disabled by default as this is the
```

Comment:
Link to the join pushdown section in the pushdown docs.

Reply:
by default.

Reply:
> Link to the join pushdown section in the pushdown docs.

Can do this with 

``` Enable :ref:`join_pushdown` ```